app/views/wizard/_form.html.haml

```diff
@@ -3,7 +3,7 @@
 -#  See the COPYING file at the top-level directory or at
 -#  https://github.com/puzzle/cryptopus.
 
-= form_tag({action: 'apply'}, id: 'wizardform') do
+= form_tag(wizard_apply_path, method: 'post', id: 'wizardform') do
```

Comment:
post is probably default, isn't it ?
https://apidock.com/rails/ActionView/Helpers/FormTagHelper/form_tag